app/javascript/packages/secret-session-storage/index.ts

```diff
@@ -0,0 +1,61 @@
+type JSONValue = string | number | boolean | null | JSONValue[] | { [key: string]: JSONValue };
+
+const ab2s = (buffer: Uint8Array) => String.fromCharCode.apply(null, new Uint8Array(buffer));
+
+export const encode = (string: string) => Uint8Array.from(string, (c) => c.charCodeAt(0));
```

Comment:
seems like these two functions are inverses of each other, should we name them to match?
-`decode`/`encode`
- `ab2s`/`s2ab`

Reply:
Good point! I'll bring them into alignment.